### PR TITLE
Setup a default set of page actions to take a scenario to the page it requires for testing.

### PR DESCRIPTION
**Pull request by mikeSmith08**
- Created a CommonSteps class which has to be in the class hierarchy so that other pages can extend from it and use its method.
 - Created a set of default actions for each page which can take in a set of overriding values, the basic navigation has a default set of values.

Still a few more navigation tests to uncomment and get working, but this gets the ball rolling and makes it possible to test each individual page now (I think). I'll finish the rest in the next PR.

I have tested this against BrowserStack too and all works well.

### CODE REVIEW

File: src/test/java/uk/gov/dhsc/htbhf/steps/CommonSteps.java [hunk 1]
```diff
@@ -1,27 +1,109 @@
 package uk.gov.dhsc.htbhf.steps;
 
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.When;
-import uk.gov.dhsc.htbhf.page.BasePage;
-import uk.gov.dhsc.htbhf.page.GuidancePage;
-import uk.gov.dhsc.htbhf.page.PageName;
+import uk.gov.dhsc.htbhf.page.*;
```
Comment (YetAnotherMatt) [inline src/test/java/uk/gov/dhsc/htbhf/steps/CommonSteps.java]: Just an observation - this class could get rather large

File: src/test/java/uk/gov/dhsc/htbhf/steps/NavigationSteps.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import static uk.gov.dhsc.htbhf.page.PageName.*;
+import static uk.gov.dhsc.htbhf.steps.Constants.DEFAULT_ACTION_OPTIONS;
+
+/**
+ * Steps that help; with navigation through the application flow.
```
Comment (dwybourn) [inline src/test/java/uk/gov/dhsc/htbhf/steps/NavigationSteps.java]: Erroneous semi-colon after help.

File: src/test/java/uk/gov/dhsc/htbhf/steps/CommonSteps.java (hunk 1 above)
Comment (mikeSmith08) [reply src/test/java/uk/gov/dhsc/htbhf/steps/CommonSteps.java]: True, the same goes for common steps in the JavaScript version as they are mirrors of each other. We can keep our eye on it and tidy up/restructure if it becomes a problem.

File: src/test/java/uk/gov/dhsc/htbhf/steps/NavigationSteps.java (hunk 2 above)
Comment (mikeSmith08) [reply src/test/java/uk/gov/dhsc/htbhf/steps/NavigationSteps.java]: Will remove in the next PR.

**Pull request merged**